Dockerfile

```
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS installer-env
+
+# Build requires 3.1 SDK
+COPY --from=mcr.microsoft.com/dotnet/core/sdk:3.1 /usr/share/dotnet /usr/share/dotnet
+
+COPY . /src/dotnet-function-app
+RUN cd /src/dotnet-function-app && \
+    mkdir -p /home/site/wwwroot && \
+    dotnet publish *.csproj --output /home/site/wwwroot
+
+FROM mcr.microsoft.com/azure-functions/dotnet:4-slim
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
```

```suggestion
ARG OWNER=pagopa
ARG REPO=repo

LABEL maintainer="https://pagopa.it"
LABEL org.opencontainers.image.source https://github.com/$OWNER/$REPO

ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
```

we need to pass $REPO var as build arg

---

https://github.com/pagopa/infra-ssl-check/pull/4/files#diff-a9b4f1a51dd81dfaef1e9d563dfe6045ab8089a05200453d6e0b2d66201f419fR37-R38